package.json

```diff
@@ -87,8 +89,11 @@
   },
   "license": "MIT",
   "scripts": {
+    "arduino:compile": "avr-pizza compile -a uno -s test/arduinoTest/arduinoEcho/arduinoEcho.ino -o test/arduinoTest/arduinoEcho",
+    "arduino:diff": "echo 'help me noopkat! && false",
```

Comment:
this is rather terribly written, but [here's a gist](https://gist.github.com/noopkat/f380a22671d2fb18f16157514219358a) as a quick example.